content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
 ---
 
 <!-- overview -->
-This page describes common concepts in the Kubernetes API.
-
+This page describes Kubernetes' API and conventions for interacting with it.
 
 <!-- body -->
 The Kubernetes API is a resource-based (RESTful) programmatic interface provided via HTTP. It supports retrieving, creating,
-updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET), includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. It also supports efficient change notifications on resources via "watches" and consistent lists to allow other components to effectively cache and synchronize the state of resources.
+updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET). The API includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. Kubernetes also supports efficient change notifications on resources via "watches" and consistent lists, so that API clients can effectively cache, track, and synchronize the state of resources.
+
+## Kubernetes API terminology {#standard-api-terminology}
```

Comment:
Why adding "Kubernetes" here and changing the title?
What else can it be if we omit "Kubernetes"?

Reply:
The previous title, “Standard API terminology”, implies some common set of principles used across a group of technologies. However, these terms are specific to Kubernetes.

Reply:
The change implies that Kubernetes does not follow RESTful standards or has a different way of representing resources?
`Kubernetes generally leverages standard RESTful terminology to describe the API concepts:`

Reply:
ReST is not a _standard_, it's more of a set of principles. Kubernetes does not closely implement those principles as laid out in the ReST paper (and that's fine).

Reply:
From a technical writing standpoint:

- @sftim is correct, REST is a set of principles, not a strictly defined standard. 
- Kubernetes follows RESTful principles as much as any other API I've ever worked with does, which is to say "mostly". 

I don't see any problem with this subtitle and I don't think further discussion should block merging this PR.